.github/fixtures/test-release-statistics/cliff.toml

```diff
@@ -0,0 +1,48 @@
+# git-cliff ~ configuration file
+# https://git-cliff.org/docs/configuration
+
+[changelog]
+# A Tera template to be rendered for each release in the changelog.
+# See https://keats.github.io/tera/docs/#introduction
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {{ commit.message | upper_first }}\
+    {% endfor %}
+{% endfor %}
+### Commit Statistics
+    * {{ statistics.commit_count }} commits contributed to the release.
+	{%- if statistics.commit_duration_days is defined %}
+	* {{ statistics.commit_duration_days }} days spanned between the first and last commit.
+	{%- endif %}
+	* {{ statistics.conventional_commit_count }} commit was understood as conventional.
+	* {{ statistics.total_link_count }} issues like '(#ID)' were seen in commit messages.
```

Comment:
Btw I'm struggling to understand how this metric is useful... can you give me an example where you'd want to see this?

Reply:
Got it.. I guess then it might be useful.

FWIW Tera has a [`length`](https://keats.github.io/tera/docs/#length) function to get the count of things.